packages/algob/test/stubs/algo-operator.ts

```diff
@@ -1,12 +1,18 @@
 
-import { Algodv2 } from "algosdk";
+import algosdk, { Algodv2 } from "algosdk";
 
 import { AlgoOperator } from "../../src/lib/algo-operator";
-import { Account, ASADef, ASADeploymentFlags, ASAInfo } from "../../src/types";
+import {
+  Account, ASADef, ASADeploymentFlags, ASAInfo, ASCCache, ASCDeploymentFlags,
+  ASCInfo
```

Comment:
Either everything has to be in new lines or nothing.

Reply:
ok